.github/actions/setup-openmetadata-test-environment/action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: Ingestion dependency to pass to run_local_docker.sh
     required: false
     default: "mysql,elasticsearch,sample-data"
+  database:
```

Comment:
Can we instead name this as `additionalArgs` instead ? Ideally this would be addtional Args for `run_local_docker.sh` right ?

Reply:
nvm. We should remove this and just use `args` inputs for adding these!

Reply:
ok fair enough. I'll just merge the current `args`